pkg/tracing/tracing.go

```diff
@@ -84,13 +100,15 @@ func (t *tracerProvider) Tracer(name string, options ...trace.TracerOption) trac
 	return t.provider.Tracer(name, options...)
 }
 
-func createTracerProvider(service string, cfg *config.Tracing) (trace.TracerProvider, error) {
+func createTracerProvider(service string, cfg *config.Tracing, user, pass string) (trace.TracerProvider, error) {
 	if !cfg.Enabled {
 		return trace.NewNoopTracerProvider(), nil
 	}
 
 	exp, err := jaeger.New(jaeger.WithCollectorEndpoint(
 		jaeger.WithEndpoint(cfg.Endpoint),
+		jaeger.WithUsername(user),
+		jaeger.WithPassword(pass),
```

Comment:
Do we pass these even if they are empty?

Reply:
It doesn't make a difference. The credentials will be ignored if there is no  auth configured